zcash_client_backend/src/lib.rs

```diff
@@ -97,7 +97,6 @@ pub enum ShieldedProtocol {
     /// The Sapling protocol
     Sapling,
     /// The Orchard protocol
-    #[cfg(zcash_unstable = "orchard")]
```

Comment:
Why is this being removed before the release that adds Orchard support?

Reply:
Same reason as previous comment.

Reply:
In that case I need to re-review this PR with an eye for everything that was behind this config flag being ready for stabilisation, because I was under the impression that _everything_ Orchard-specific was only being stabilised in the release after next.